core-ui/src/components/Predefined/Details/Service/Service.details.scss

```diff
@@ -1,10 +1,10 @@
-.service__port-container.fd-token {
+.fd-token {
```

Comment:
Please don't overwrite fundamental styles globally

Reply:
You can leave it as `.service-ports.fd-token {` and add class `className="service-ports" ` to the `<Token>`